App.js

```diff
@@ -77,6 +83,31 @@ class App extends Component {
     LayoutAnimation.spring();
   }
 
+  getCurrentRouteName(navigationState) {
+    if (!navigationState) {
+      return null;
+    }
+    const route = navigationState.routes[navigationState.index];
+
+    if (route.routes) {
+      return this.getCurrentRouteName(route);
+    }
+
+    return route.routeName;
+  }
+
+  statusBarHandler(prev, next) {
+    const routeName = this.getCurrentRouteName(next);
+
+    const { translucent, backgroundColor, barStyle } = getStatusBarConfig(
+      routeName
+    );
+
+    StatusBar.setTranslucent(translucent);
+    StatusBar.setBackgroundColor(backgroundColor(routeName));
```

Comment:
@jouderianjr Please return `backgroundColor` value, instead of a function. Except here everything is so good!

Reply:
`backgroundColor` in lightStatusBar depends of if you are in `Login` or not. But I have an Idea, I can transform `lightStatusBar` and `darkStatusBar` in a function that receives the routeName and return the object.

Example: 

```
const lightStatusBar = routeName => {
  translucent: true,
  backgroundColor: routeName === 'Login' ? colors.transparent : colors.black,
  barStyle: 'light-content',
};
```

What do you think about?

Reply:
Right, that's what I want.

Reply:
And you may rename `lightStatusBar` as `getLightStatusBar`.